exemplar/src/main/kotlin/com/squareup/exemplar/ExemplarModule.kt

```diff
@@ -15,5 +16,6 @@ class ExemplarModule : AbstractModule() {
         install(WebActionModule.create<ReadinessCheckAction>())
         install(WebActionModule.create<LivenessCheckAction>())
         install(WebActionModule.create<NotFoundAction>())
+        install(WebActionModule.create<MetricsAction>())
```

Comment:
This should be before `NotFoundAction`. Maybe we should warn if `NotFoundAction` isn't last?

Reply:
Yes, seem fragile for binding statements to have an implicit required ordering. I'd hard error if the required ordering is not correct. Same for interceptors.